pkgs/development/python-modules/mitmproxy-rs/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, rustPlatform
+, darwin
+}:
+
+buildPythonPackage rec {
+  pname = "mitmproxy-rs";
```

Comment:
If we go really strongly by the contributing guide then this should have been it's own commit but it's alright for me if I am honest :)